AmericanMedicalAssociation/AMA-style-guide
# Ewl 3072 styles automation

Ticket: https://issues.ama-assn.org/browse/EWL-3072

## Testing instructions
* `npm install` 
* `gulp serve`
*  change some sass
* Observe autoprefixer runs
* Observe indentation is updated
* `gulp scss-lint`
* Observe there is linting about the quotes, leading zeros, and indentation as well as other items

_does not include linting for Twig, since I was unable to find a way to do that_
